### PR TITLE
Suppress this-escape warnings

### PR DESCRIPTION
**Pull request by benwtrent**
making JDK21+ happy

### CODE REVIEW

File: x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/TokenService.java [hunk 1]
```diff
@@ -2396,6 +2397,7 @@ private void submitUnbatchedTask(@SuppressWarnings("SameParameterValue") String
         clusterService.submitUnbatchedStateUpdateTask(source, task);
     }
 
+    @SuppressWarnings("this-escape")
```
Comment (benwtrent) [inline x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/TokenService.java:2400]:
```suggestion
```

**Comment by benwtrent** [comment]
@elasticmachine update branch

**Review by dnhatn — APPROVED** [review]
LGTM.

**Comment by benwtrent** [comment]
@elasticmachine update branch

**Pull request merged**